static/src/javascripts/projects/common/modules/accessibility/main.spec.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import { saveState, isOn } from 'common/modules/accessibility/main';
+
+jest.mock('common/modules/user-prefs', () => {
+    const storage = {};
+
+    return {
+        get(key) {
+            return storage[key];
+        },
+        set(key, value) {
+            storage[key] = value;
+        },
+    };
+});
```

Comment:
What do you think about moving this into `__mocks__`, since it could be potentially interesting for other tests aswell?

Reply:
When I was writing it, I thought the same thing! Good shout, let's do it 👍